internal/resource/config/oauth/oauth_auth_server_settings_scopes_exclusive_scope_data_source.go

```diff
@@ -0,0 +1,113 @@
+package oauth
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	client "github.com/pingidentity/pingfederate-go-client/v1125/configurationapi"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/datasource/common/id"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/config"
+	internaltypes "github.com/pingidentity/terraform-provider-pingfederate/internal/types"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ datasource.DataSource              = &oauthAuthServerSettingsScopesExclusiveScopesDataSource{}
+	_ datasource.DataSourceWithConfigure = &oauthAuthServerSettingsScopesExclusiveScopesDataSource{}
+)
+
+// Create a Administrative Account data source
+func NewOauthAuthServerSettingsScopesExclusiveScopesDataSource() datasource.DataSource {
+	return &oauthAuthServerSettingsScopesExclusiveScopesDataSource{}
+}
+
+// oauthAuthServerSettingsScopesExclusiveScopesDataSource is the datasource implementation.
+type oauthAuthServerSettingsScopesExclusiveScopesDataSource struct {
+	providerConfig internaltypes.ProviderConfiguration
+	apiClient      *client.APIClient
+}
+
+type oauthAuthServerSettingsScopesExclusiveScopesDataSourceModel struct {
+	Id          types.String `tfsdk:"id"`
+	Name        types.String `tfsdk:"name"`
+	Description types.String `tfsdk:"description"`
+	Dynamic     types.Bool   `tfsdk:"dynamic"`
+}
+
+// GetSchema defines the schema for the datasource.
+func (r *oauthAuthServerSettingsScopesExclusiveScopesDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	schemaDef := schema.Schema{
+		Description: "Manages a OauthAuthServerSettingsScopesCommonScopes.",
```

Comment:
I haven't noticed this on previous data sources, but "Manages" is probably not the right language to use for a data source, since data sources can't modify anything. I used "Describes" in the PD provider.